grafana-dashboards/grafana-dashboard-terraform-repo.yaml

```diff
@@ -142,7 +191,7 @@ data:
                 "uid": "P1A97A9592CB7F392"
               },
               "dimensions": {},
-              "expression": "fields @timestamp, message |\nfilter kubernetes.pod_name like /tf-repo-push-deploy-pipelinerun.+/ |\nfilter kubernetes.container_name not like /place-scripts|prepare|working-dir-initializer/ |\nfilter message not like \"using gql endpoint\" | \nfilter message not like \"No changes need to be made\" |\n sort @timestamp desc | limit 400",
+              "expression": "fields @timestamp, message, kubernetes.labels.tekton_dev_pipelineRun |\nfilter kubernetes.pod_name like /tf-repo-push-deploy-pipelinerun.+/ |\nfilter kubernetes.container_name not like /place-scripts|prepare|working-dir-initializer/ |\nfilter kubernetes.labels.tekton_dev_pipelineRun like /$pipelinerun/ |\nfilter message not like \"using gql endpoint\" | \nfilter message not like \"No changes need to be made\" |\n sort @timestamp desc | limit 400",
```

Comment:
should we sort by `@timestamp asc` for terraform logs, like in https://github.com/app-sre/qontract-reconcile/pull/4857

Reply:
+1. Let's also remove the `limit 400` we want the full log of the given pipelinerun. 

I think I would also not filter out `No changes need to be made`. This is a valuable info IMHO

Reply:
Updated